kore/src/Kore/Syntax/Json.hs

```diff
@@ -6,11 +6,15 @@ License     : BSD-3-Clause
 -}
 module Kore.Syntax.Json (
     -- API
+    KoreJson,
     JsonParseError,
     encodePattern,
     encodeKoreJson,
     decodePattern,
     decodeKoreJson,
+    toParsedPattern,
+    term,
+    fromPattern,
 ) where
```

Comment:
@jberthold I'm re-exporting some of the internals here. Hope that's ok?

Reply:
No problem at all, makes sense to export the conversion functions. I would maybe 
* export `KoreJson(..)` rather than `term` 
* also export `KorePattern` (type only, without constructors)
but fine either way